src/_base/_twig/docker-compose.yml/service/nginx.yml.twig

```diff
@@ -3,7 +3,7 @@
 {% set syncvolume = true %}
 {% endif %}
 {% set hostnames = [@('hostname')] %}
-{% set hostnames = hostnames|merge(@('hostname_aliases')|map(alias => "#{alias}." ~ @('domain'))) %}
+{% set hostnames = hostnames|merge(@('hostname_aliases')| filter(v => v is not empty) | map(alias => "#{alias}." ~ @('domain'))) %}
```

Comment:
Just this bit now that is what we want to keep after @Chemaclass's PRs now merged

Reply:
https://github.com/inviqa/harness-base-php/pull/689, I've instead done at the source